config/kubernetes/sig-docs/teams.yaml

```diff
@@ -29,8 +29,8 @@ teams:
     - xiangpengzhao
     - zhangxiaoyu-zidif
     privacy: closed
-  sig-docs-fr-reviews:
-    description: Review PRs for French content
+  sig-docs-fr-owners:
```

Comment:
@rlenferink While we are here, can you add @abuisine to fr-owners and fr-reviewers?

Ref: https://github.com/kubernetes/org/pull/534#issuecomment-468235782

Reply:
Done: https://github.com/kubernetes/org/pull/546/commits/1878a0e835f8ff696493f1548106c2100126b685